src/Tvheadend.cpp

```diff
@@ -598,6 +602,43 @@ PVR_ERROR CTvheadend::RenameRecording ( const PVR_RECORDING &rec )
   return SendDvrUpdate(m);
 }
 
+PVR_ERROR CTvheadend::SetPlayCount ( const PVR_RECORDING &rec, int playcount )
+{
+  if (m_conn.GetProtocol() < 26)
+    return PVR_ERROR_NOT_IMPLEMENTED;
+
+  Logger::Log(LogLevel::LEVEL_DEBUG, "Setting playcount to %i for recording %s", playcount, rec.strRecordingId);
+
+  /* Build message */
+  htsmsg_t *m = htsmsg_create_map();
+  htsmsg_add_u32(m, "id",        atoi(rec.strRecordingId));
+  htsmsg_add_u32(m, "playcount", playcount);
+  return SendDvrUpdate(m);
+}
+
+PVR_ERROR CTvheadend::SetPlayPosition ( const PVR_RECORDING &rec, int playposition )
+{
+  if (m_conn.GetProtocol() < 26)
+    return PVR_ERROR_NOT_IMPLEMENTED;
+
+  Logger::Log(LogLevel::LEVEL_DEBUG, "Setting playposition to %i for recording %s", playposition, rec.strRecordingId);
+
+  /* Build message */
+  htsmsg_t *m = htsmsg_create_map();
+  htsmsg_add_u32(m, "id",           atoi(rec.strRecordingId));
+  htsmsg_add_u32(m, "playposition", playposition);
+  return SendDvrUpdate(m);
+}
+
+int CTvheadend::GetPlayPosition ( const PVR_RECORDING &rec )
+{
+  if (m_conn.GetProtocol() < 26)
+    return PVR_ERROR_NOT_IMPLEMENTED;
+
+  /* Seems a bit odd */
```

Comment:
Kodi calls this function to request the play position, although the play position was already transferred to Kodi as a member of PVR_RECORDING.